src/engine/server/server.cpp

```diff
@@ -52,6 +53,12 @@
 	#include <windows.h>
 #endif
 
+// uncomment the following line to measure server performance
+//#define MEASURE_TICKS
```

Comment:
to activate it you have to uncomment this line, maybe there is a better way of doing this?

Reply:
#define is a preprocessor directives. They can be controlled at compile time. Check out -D option of gcc - https://www.rapidtables.com/code/linux/gcc/gcc-d.html

Reply:
I am using bam to compile, do you know how to define it over bam?

Reply:
Ok i found out you can add this line to the bam.lua file

> debug_settings.cc.defines:Add("MEASURE_TICKS")

i put it after this line: https://github.com/yavl/teeworlds-infclassR/blob/master/bam.lua#L348